src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -22,10 +22,12 @@
                 ></a>
             </span>
         </div>
-        <div class="main-image s-image-main c-imagery__main-image has-local-controls"
+        <div class="main-image s-image-main c-imagery__main-image has-local-controls j-imagery-image"
```

Comment:
Can we rename this js-imageryView-image? My bad I think

Reply:
haha, I tried to clear it with you! Updated : )

Reply:
@charlesh88 Pointed out that the correct prefix is actually `js-`, not `j-`. I messed up in my original naming suggestion.